src/templates/shared/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    {% include "includes/head.html" %}
+    <body>
+        {% include "includes/header.html" %}
+        <div class="container">
+            <div class="main">
+                <h2>1. Browse journals</h2>
+                <div class="box">
+                    {% include "includes/section-search.html" %}
+                </div>
+
+                <h2>2. View and customise feeds</h2>
```

Comment:
I see what you're doing here - I think that we can structure the page without steps by putting the download button at the top, and when we implement any sort of query customization doing that by having the table rows expand to show options :). I think we should make separate pages for previewing the feeds themselves and then make the customization panel a partial we can show both in the expanded table row and the journal feed page.

Does that make sense? if so it it alright if i take out sections 2. and 3. and move the (currently disconnected) export button to the top?

Reply:
I'm not 100% sure I follow. Is it a question about the 1 vs. 2 tables?

If we have a single table only, then what happens when the user chooses one item, then searches for a second? Then that query either sticks to the top or disappears - after a couple of selections, sticky will be quite annoying. 
Also, it means that if someone wanted `eLife+keyword:brain` and `eLife+keyword:gut` as two separate feeds, that becomes very tricky.

So I suggest a 2 table solution, but we can also have a preview button so one can preview with the tags (or without), as I do agree that's quite a nice feature. :)

Reply:
But we could move the button up to be part of the second table, similar place to the search bar in the first table?

Reply:
If you haven't seen my latest reply to the discussion, I talked a lot about it in there: https://github.com/sneakers-the-rat/journal-rss/discussions/2#discussioncomment-8001718

Reply:
aha i get what you're saying - see i wasn't really imaging it as an interface where someone would load up like 20 feeds at once or w/e, but do that in multiple exports, but the "two feeds from the same journal with different query parameters" got me to see what you're talking about.

my hesitation with the two tables is mostly that it seems like we would need to either have a lot of dead space to keep both of them visible, or else if the second table was offscreen it wouldn't be clear that pressing the button was adding that row to that table. But I think that could be fixed by making the top table have flexbox expand to fill space, and the second table being min-height'ed to be one-row tall at the bottom, visually distinct. I feel like the dynamics of interacting with that would make it so we don't need to burn space on the mid-page instruction titles, but let me give it a try.

in the meantime i'm good with this

Reply:
Aaaah, haha, that explains it then! :D 
I think the ability to generate a list of feeds would be a massive thing - rather than having to repeat and import multiple feeds - that's what I tried to implement in https://github.com/roaldarbol/rss-journals (instance on https://roaldarbol.shinyapps.io/rss-journals/), but staled at the lack of all amazing work you've done here. 

We could also think about having two separate pages whilst fidgeting about, one for single export and one for multi export? Might give us some room for trial and error.

But I like your suggestions re UX!

Reply:
i'm not too picky here, ux is something that i like to iterate around with a lot rather than preoptimizing ideas out of existence. still need to make a bunch of core functionality stuff before we worry about that too much, feel free to play around :)